TimedArithmeticQuiz/src/App.jsx

```diff
@@ -0,0 +1,66 @@
+import { useCallback, useEffect, useRef, useState } from "react"
+import { QuizHeader } from "./components/QuizHeader"
+import { quizConfig } from "./data/quizConfig"
+import { Question } from "./components/Question";
+import { Quiz } from "./models/Quiz";
+import { QuestionHeader } from "./components/QuestionHeader";
+import { QuizComplete } from "./components/QuizComplete";
+
+
+function App() {
+  const [score, setScore] = useState(0);
+  const [quiz,] = useState(new Quiz(quizConfig.numberRange, quizConfig.operatrs ,quizConfig.totalQuestionCount));
+  const [currentQuestionIndex, setCurrrentQuestionIndex] = useState(0);
+  const [enteredAnswer, setEnteredAnswer] = useState("");
+
+  const initialized = useRef(false);
+  useEffect(() => {
+    if (initialized.current) return;
+    initialized.current = true;
+    
+    quiz.startQuiz();
+    setCurrrentQuestionIndex(quiz.getCurrentIndex());
+  }, []);
+  
+
+  const onNextClick = useCallback(function onNextClick(timedOut = false) {
+    setEnteredAnswer(answer => {
+      if (quiz.isResponseForCurrentQuestionCorrect(answer, timedOut)) {
+        setScore(s => s+1);
+      }
+      return "";
+    });
+    quiz.nextQuestion();
+    setCurrrentQuestionIndex(quiz.getCurrentIndex());
+  }, []);
+
+  let body;
```

Comment:
Rather than having a variable, we can just return either option wherever applicable